holiday-management/src/controllers/holiday-controller.ts

```diff
@@ -140,9 +140,7 @@ router.post('/api/holidays', (req: Request, res: Response, next: express.NextFun
         return next(new ResponseError('Container for this user already exists', 400));
       }
     })
-    .catch(err => {
-      return next(err);
-    });
+    .catch(err => next(err));
```

Comment:
It can be even simplified to `catch(next)` :)

Reply:
Simplified